yarn-project/end-to-end/src/flakey_e2e_account_init_fees.test.ts

```diff
@@ -1,11 +1,14 @@
 import { getSchnorrAccount } from '@aztec/accounts/schnorr';
```

Comment:
I wanted to include some code of this test in docs and I felt that the use of ctx instead of destructuring made it harder to read so I change that.